filament/src/ShadowMap.cpp

```diff
@@ -1128,13 +1128,13 @@ void ShadowMap::visitScene(const FScene& scene, uint32_t visibleLayers,
     SYSTRACE_CALL();
 
     using State = FRenderableManager::Visibility;
-    FScene::RenderableSoa const& UTILS_RESTRICT soa = scene.getRenderableData();
```

Comment:
Why remove the restricts here?

Reply:
Ah yes, they were useless because we're only reading from these pointers. I don't know why I did that.